scripts/tests/test_tractogram_fix_trk.py

```diff
@@ -3,5 +3,5 @@
 
 
 def test_help_option(script_runner):
-    ret = script_runner.run('scil_convert_tractogram.py', '--help')
+    ret = script_runner.run('scil_tractogram_fix_trk.py', '--help')
     assert ret.success
```

Comment:
In version 1.6., there was no more testing than the help?

Reply:
It's complicated to find example of bad trk :(  I will open an issue